### ISSUE TITLE
UNETR‘s training accuracy is  always below value in the paper 

### ISSUE THREAD

**Issue by aubest** [Project-MONAI/tutorials#642]
Hi, I attempted to train UNETR, but I found the accuracy value always at about 0.80.
I know the Vit was used in the encoder,  Do I need to load the  pre-training model when I use it?
What can I do for training to make  the accuracy value improve?

**Comment by achen46** [comment]
Hi @aubest , values reported in the paper are from 5 fold cross-validation, hence being better than any single fold results. In addition, we use a cohort of 80 images, which come from a privately-held dataset, for training. Please see section 4.3. 

I recommend using the pre-trained weights as provided in the Research Contribution Repository for fine-tuning.

**Comment by aubest** [comment]
@Nic-Ma,  Thank you very much!

**Comment by aubest** [comment]
Hi@ahatamiz @achen46, Thanks for your reply. Your advice is very useful and it has enlightened me. I will try to train it again. I am hoping to communicate with you in the future. Thank you！
